src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -148,13 +148,15 @@ export function addStatusToIssues(issues) {
 }
 
 /**
- * Finds an appeal from the Redux store with ID matching arg ID
+ * Finds an appeal from the Redux store with ID matching arg ID.
+ * `id` may be a v1 id or a v2 id.
+ *
  * @param {object} state Full redux store state tree
  * @param {string} id Appeal ID of the appeal to find
  * @returns {object} One appeal object or undefined if not found in the array
  */
-export function isolateAppeal(state, id) {
-  return _.find(state.disability.status.claimsV2.appeals, (a) => a.id === id);
+export function isolateAppeal(state, id, v1ToV2IdMap) {
+  return _.find(state.disability.status.claimsV2.appeals, (a) => a.id === id || v1ToV2IdMap[id]);
```

Comment:
This is great as is. Is if of interest to use the map to store the appeal as the value of the key above, so that we can have O(1) lookup here? If so, we could also store the v2 ids and appeals in the same structure as well (but probably with a new name).

Reply:
Yeah, great point. We mostly copy-pasta'd this from v1 without trying to make it efficient. That would be a good candidate for refactor (which we will have a lot of if we can ever get around to addressing tech debt).